### PR TITLE
Enhancement/remove unit test errors

### LINKED ISSUE #1321: [Housekeeping] Remove console errors in unit tests
**Is your enhancement request related to a problem? Please describe.**
Errors in console while running our specs

**Describe the solution you'd like**
Fix the issues which are causing the error logs by replacing all imports of IonicModule with `TestHelper.ionicModuleForTest`

**Additional context**
see example: https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.spec.ts#L15

https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/testing/test-helper.ts

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
This PR closes #1321 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement (to existing content)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
There are a lot of `config.get` errors that surface when running the test suite in the browser. The `TestHelper` method `ionicModuleForTest` implements a fix for this.

## What is the new behavior?
Everywhere spec files use `IonicModule.forRoot({ mode: 'ios', _testing: true })`, we now use `TestHelper.ionicModuleForTest`. 

Additionally, a few missing components are now declared to minimize the amount of warnings in the console when running the tests. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: libs/designsystem/src/lib/components/modal/services/action-sheet.helper.spec.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Component } from '@angular/core';
 import { IonicModule, ModalController as IonicModalController } from '@ionic/angular';
```
Comment (MadsBuchmann) [inline libs/designsystem/src/lib/components/modal/services/action-sheet.helper.spec.ts:2]:
```suggestion
import { ModalController as IonicModalController } from '@ionic/angular';
```

Consider removing unused import 😀

**Review by jakobe — APPROVED** [review]
LGTM!

**Review by MadsBuchmann — CHANGES_REQUESTED** [review]

File: libs/designsystem/src/lib/components/modal/services/action-sheet.helper.spec.ts (hunk 1 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/src/lib/components/modal/services/action-sheet.helper.spec.ts:2]: Fixed in bc24f2804b8bc3bb07d8c74b668286fbcff9aa98

File: libs/designsystem/src/lib/components/avatar/avatar.component.spec.ts [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-
-import { IconComponent } from '../icon/icon.component';
-import { AvatarComponent } from './avatar.component';
-import { SizeDirective } from '../../directives/size/size.directive';
 import { ThemeColorDirective } from '../../directives';
+import { SizeDirective } from '../../directives/size/size.directive';
 import { ColorHelper } from '../../helpers';
-import { ProgressCircleComponent } from '../progress-circle/progress-circle.component';
-import { ProgressCircleRingComponent } from '../progress-circle/progress-circle-ring.component';
 import { styles } from '../../helpers/color-helper.styles';
```
[the remark this answers is not in the document]
Comment (RasmusKjeldgaard) [reply libs/designsystem/src/lib/components/avatar/avatar.component.spec.ts]: Fixed in bc24f2804b8bc3bb07d8c74b668286fbcff9aa98

File: libs/designsystem/src/lib/components/button/button.component.integration.spec.ts [hunk 3]
```diff
@@ -33,7 +33,7 @@ describe('ButtonComponent in Kirby Page', () => {
   let spectator: SpectatorHost<PageComponent>;
   const createHost = createHostFactory({
     component: PageComponent,
-    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true }), RouterTestingModule],
+    imports: [TestHelper.ionicModuleForTest, RouterTestingModule],
     declarations: [
```
[the remark this answers is not in the document]
Comment (RasmusKjeldgaard) [reply libs/designsystem/src/lib/components/button/button.component.integration.spec.ts:37]: Fixed in bc24f2804b8bc3bb07d8c74b668286fbcff9aa98

**Review by MadsBuchmann — APPROVED** [review]
LGTM 🚀

**Pull request merged**